__tests__/app.test.js

```diff
@@ -17,18 +32,19 @@ describe("GET /api/", () => {
                 description: expect.any(String)
             };
             expect(status).toBe(200);
+            expect(topics.length).not.toBe(0);
             topics.forEach((topics) => {
                 expect(topics).toMatchObject(expected);
 
             });
 
         });
-        test("404: status code when url path is not found",async() => {
-            const {status,body} = await request(app).get("/api/topucs")
-            
-            expect(status).toBe(404)
-            expect(body.msg).toBe("Path not found")
-        })
-        
+        test("404: status code when url path is not found", async () => {
+            const { status, body } = await request(app).get("/api/topucs");
+
```

Comment:
This 404 test isn't specific to this endpoint. You can put this in a separate describe block 😄